test/DynamoCoreTests/UnitTestBase.cs

```diff
@@ -64,7 +67,10 @@ public static string TestDirectory
         [SetUp]
         public virtual void Setup()
         {
-            System.Console.WriteLine("Start test: " + TestContext.CurrentContext.Test.Name);
+            
+            System.Console.WriteLine($"PID {Process.GetCurrentProcess().Id} Start test: {TestContext.CurrentContext.Test.Name}");
+            TestUtilities.WebView2Stamp = TestContext.CurrentContext.Test.Name;
```

Comment:
ah I see... guess it's not a big deal but just to offer an alternative, I think you could store the stack trace of the current code that is calling the constructor and you might be able to get this same info - this is simpler though.

Reply:
We can definitely add more info (like the stack). But for now I did not find it useful